src/main/java/seedu/address/logic/parser/SortCommandParser.java

```diff
@@ -20,7 +20,7 @@
 public class SortCommandParser implements Parser<SortCommand> {
 
     public static final String PARSE_EXCEPTION_MESSAGE = "Invalid syntax for sort. Please use the "
-            + "following syntax in []: sort [/byname][/byemail][/byphone][/byaddress] (Optional)[/reverse]";
+            + "syntax as stated in user guide. For example: sort /byname /reverse";
```

Comment:
I think the initial explanation is better. It would be better to not have the user open the user guide whenever they need a reminder of what to input.